data/dialog phrases.txt

```diff
@@ -39,6 +39,9 @@ phrase "generic cargo delivery payment"
 phrase "generic cargo on visit"
 	word
 		`You have reached <planet>, but not all of the cargo is in the system! Better depart and wait for your escorts to arrive in this star system.`
+phrase "generic cargo with stopover on visit"
```

Comment:
This phrase assumes that you're picking up the cargo from the stopover. While this works for the missions you're using it on, I think it should be made clear from the phrase name that the stopover is the pickup location for the cargo.
```suggestion
phrase "generic cargo with stopover pickup on visit"
```